src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,29 @@
 <html>
 	<head>
 		<script>
+
+			window.gitpodIDEErrors = []
+
+			const onVsCodeWorkbenchError = (event) => {
+				const flags = new Map();
+
+				// We take a look at what is the resource that was attempted to load;
+				const resourceSource = event?.target?.getAttribute('src') || event?.target?.getAttribute('href');
+
+				// If the event has a `target`, it means that it wasn't a script error
+				if (resourceSource) {
+					if (resourceSource.match(new RegExp(/\/build\/ide\/code:.{1,100}\/__files__\//g))) {
```

Comment:
here it is not necessary, we don't fetch it from anything else, just always put 'vscode-web-workbench'